src/base64io/__init__.py

```diff
@@ -273,8 +273,12 @@ def read(self, b=-1):
         data = self.__wrapped.read(_bytes_to_read)
         # Remove whitespace from read data and attempt to read more data to get the desired
         # number of bytes.
-        if any([char.encode("utf-8") in data for char in string.whitespace]):
-            data = self._read_additional_data_removing_whitespace(data, _bytes_to_read)
+        if isinstance(data, bytes):
```

Comment:
We have some heavy duplication between these branches. What if we just prepare the whitespace characters in the `if` block, then use them outside?

```python
if isinstance(data, bytes):
    whitespace = [char.encode("utf-8") for char in string.whitespace]
else:
    whitespace = string.whitespace

if any([char in data for char in whitespace]):
    data = self._read_additional_data_removing_whitespace(data, _bytes_to_read)
```

Reply:
Done as suggested. Thanks!